port/common/omrsysinfo_helpers.c

```diff
@@ -33,7 +33,7 @@
 
 #include <string.h>
 
-#if (defined(J9X86) || defined(J9HAMMER))
+#if (defined(OMR_OS_WINDOWS) || defined(J9X86) || defined(J9HAMMER))
```

Comment:
while we're here, might as well remove the redundant parens:
```suggestion
#if defined(OMR_OS_WINDOWS) || defined(J9X86) || defined(J9HAMMER)
```